src/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useContext } from 'react';
+import classNames from 'classnames';
+import { DispatchContext } from '../TodosContext';
+
+export const Header: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [isHasError, setIsHasError] = useState(false);
+  const onSubmit = useContext(DispatchContext);
+  const handleChandge = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setIsHasError(false);
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    setIsHasError(!title);
+
+    if (title) {
+      onSubmit({
+        type: 'add',
+        payload: {
+          id: +new Date(),
+          title,
+          completed: false,
+        },
+      });
+      setTitle('');
+    }
+  };
+
+  return (
+    <header className="header">
+      <h1>todos</h1>
+
+      <form onSubmit={handleSubmit}>
+        <input
+          // eslint-disable-next-line jsx-a11y/no-autofocus
+          autoFocus
+          type="text"
+          data-cy="createTodo"
+          className={classNames(isHasError
+            ? 'new-todo error'
+            : 'new-todo')}
+          placeholder={classNames(isHasError
```

Comment:
Do you really needs `classNames` here?

Reply:
to show error in input and change it, if it have